hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -189,101 +189,62 @@ msgid "Support / Contact"
-"Bitte folge diesem Link und wähle ein neues Passwort: %(base_url)s"
-"%(password_reset_url)s Solltest du kein neues Passwort angefordert haben, "
-"dann ignoriere diese E-Mail.<br/>\n"
-"Dankeschön!\n"
 
-#, python-format
-msgid ""
-"You're receiving this email because you requested a password reset for your "
-"user account at %(site_name)s.\n"
-"Please go to the following page and choose a new password: %(base_url)s"
-"%(password_reset_url)s\n"
-"If you didn't request a new password, ignore this e-mail.\n"
-"Thank you!\n"
-msgstr ""
-"Du erhälst diese E-Mail da du dein Passwort für deinen Account bei "
-"%(site_name)s zurücksetzen möchtest.\n"
-"Bitte folge diesem Link und wähle ein neues Passwort: %(base_url)s"
-"%(password_reset_url)s Solltest du kein neues Passwort angefordert haben, "
-"dann ignoriere diese E-Mail.\n"
-"Dankeschön!\n"
+msgid "If you didn't make this request you can safely ignore this email."
+msgstr ""
 
-#, python-format
-msgid ""
-"You're receiving this mail because your virtual machine [%(vm_name)s] has "
-"been cancelled.<br/>\n"
-"You can see your order status by clicking [my VM page] below.<br/>\n"
-"If you want to order a new virtual machine, you can do it by clicking <a "
-"href=\"%(base_url)s%(my_virtual_machines_url)s\">this link</a>.<br/>\n"
+msgid "Otherwise, click here to reset your password."
 msgstr ""
-"Du erhälst diese E-Mail, da deine virtuelle Maschine [%(vm_name)s] gekündigt "
-"wurde.<br/>\n"
-"Um deinen Auftragsstatus zu sehen, klicke auf die [my VM page] unten.<br/>\n"
-"Falls du eine neue virtuelle Maschine bestellen möchtest, kannst du dies "
-"tun, indem du <a href=\"%(base_url)s%(my_virtual_machines_url)s\">diesen "
-"Link klickst</a>.<br/>\n"
 
-msgid "My VM page"
+msgid "Thank you!"
 msgstr ""
 
+msgid "Virtual Machine Cancellation"
+msgstr "VM Kündigung"
+
 #, python-format
 msgid ""
-"You're receiving this mail because your virtual machine [%(vm_name)s] has "
-"been cancelled.\n"
-"You can see your order status by clicking here\n"
-"%(base_url)s%(vm_order_url)s\n"
-"If you want to order a new virtual machine, you can do it by clicking this "
-"link.\n"
-"%(base_url)s%(my_virtual_machines_url)s\n"
-msgstr ""
-"Du erhälst diese E-Mail, da deine virtuelle Maschine [%(vm_name)s] gekündigt "
-"wurde.\n"
-"Um deinen Auftragsstatus zu sehen, klicke hier.\n"
-"%(base_url)s%(vm_order_url)s\n"
-"Falls du eine neue virtuelle Maschine bestellen möchtest, kannst du dies "
-"tun, indem du diesen Link klickst.\n"
-"%(base_url)s%(my_virtual_machines_url)s\n"
+"You are receiving this email because your virutal machine [%(vm_name)s] has "
+"been cancelled."
+msgstr ""
+"Du erhälst diese E-Mail, da deine virtuelle Maschine [%(vm_name)s] "
+"gekündigt wurde."
+
+msgid "You can always order a new VM by clicking the button below."
+msgstr ""
+
+msgid "CREATE VM"
+msgstr "NEUE VM"
+
+msgid "You can always order a new VM by following the link below."
+msgstr ""
 
 msgid "Toggle navigation"
```

Comment:
Where do we have "toggle navigation"??

Reply:
its for screen readers for top navigation toggle button